From 93fcaf25e0bfb22b19c8c2f1727de4555d18267d Mon Sep 17 00:00:00 2001
From: Sravya Tirukkovalur <sravya@cloudera.com>
Date: Mon, 12 Sep 2016 09:58:32 -0500
Subject: [PATCH 0765/1164] CDH-45117 HIVE-14011: MessageFactory is not
 pluggable (Sravya Tirukkovalur, reviewed by Mohit
 Sabharwal)

(cherry picked from commit e09f3c7a0c641c580483eb52f1f91bce8a0828c7)

Change-Id: I7974974d1d4bff61b4e09437b4f52cc6fc4d13b6
---
 .../hive/hcatalog/messaging/MessageFactory.java    |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hcatalog/server-extensions/src/main/java/org/apache/hive/hcatalog/messaging/MessageFactory.java b/hcatalog/server-extensions/src/main/java/org/apache/hive/hcatalog/messaging/MessageFactory.java
index 8937745..60efba7 100644
--- a/hcatalog/server-extensions/src/main/java/org/apache/hive/hcatalog/messaging/MessageFactory.java
+++ b/hcatalog/server-extensions/src/main/java/org/apache/hive/hcatalog/messaging/MessageFactory.java
@@ -37,7 +37,7 @@
  */
 public abstract class MessageFactory {
 
-  private static MessageFactory instance = new JSONMessageFactory();
+  private static MessageFactory instance = null;
 
   protected static final HiveConf hiveConf = new HiveConf();
   static {
-- 
1.7.9.5

